chapters/curate-maintain/content/reading/README.md

```diff
@@ -1,25 +1,38 @@
 ### Content
 
-Owner: Adrian Șendroiu
-Reviewer: Ștefan Jumărea
+The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
 
-typically Git & GitHub
+The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.
+It is also recommended using a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
 
-versioning, maintaining up-to-date
+The content should be published under a license.
+At `Open Education Hub` we employ the `Creative Commons` license ourselves and strongly recommend its usage.
 
-encouraging contributions, integrating changes
+#### Contributions
 
-creating issues, monitoring issues and pull requests
+The content maintainer should foster a welcoming environment for outside contributions.
+This can be achieved by:
 
-licensing, ownership
+- Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
+- Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
+- Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)
 
-the process of creating a repository from zero
-how the structure should be created;
-initial contributions;
-submission via PRs and review process
+There should also be an established process for handling contributions whenever they arrive.
+Some steps that the maintainer should take are:
 
-the process of improving a repository via contributions;
-how the contribution + review process should happen;
-steps to be done, reviewers, assignees
+- Review the contribution as soon as possible.
+Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
+Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
+- Engage in a constructive discussion with the contributor to clarify any questions or concerns.
+Provide feedback and guidance for improvement if the contribution needs changes.
+- Ensure that the contribution follows the guidelines.
+This can be partially automated using linters and automated checkers, together with features like `GitHub Actions`.
+- If applicable, test the contribution to ensure it works as intended.
+- Thank the contributor for their effort.
```

Comment:
Maybe add that the maintainer should make sure contributions are merged, and do not end up lingering for too long?